Cargo.toml

```diff
@@ -122,7 +122,7 @@ dice-mfg-msgs = { git = "https://github.com/oxidecomputer/dice-util", default-fe
 gateway-messages = { git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["smoltcp"] }
 hif = { git = "https://github.com/oxidecomputer/hif", default-features = false }
 humpty = { git = "https://github.com/oxidecomputer/humpty", default-features = false }
-hubtools = { git = "https://github.com/oxidecomputer/hubtools", default-features = false }
+hubtools = { git = "https://github.com/oxidecomputer/hubtools", default-features = false, branch = "bin-mode" }
```

Comment:
So, by using a branch here, we're committing to retain that branch forever or the commits between here and any future switch to a `main` branch become unreproduceable. Comfortable with that?

Reply:
I'm planning to switch back to `main` once the `hubtools` PR is merged!